modules/internal/CPtr.chpl

```diff
@@ -70,6 +70,7 @@ module CPtr {
   pragma "no object"
   pragma "no default functions"
   pragma "no wide class"
+  pragma "use default init"
```

Comment:
I'm assuming there is another type with "no object" applied to it that can't use default initializers.  Otherwise, it seems like "no object" shouldn't be special cased as you have earlier.  Could that type be using "no use default init" instead, so that "no object" doesn't have to be special cased and these three types don't need the additional pragma?

Reply:
Discussed offline, but responding for posterity: The changes involving FLAG_NO_OBJECT avoid dealing with inheritance and 'super' fields, which don't apply to types with that flag.